src/main/java/com/jvm_bloggers/core/newsletter_issues/PublishingConstants.java

```diff
@@ -2,6 +2,10 @@
 
 class PublishingConstants {
 
+    private PublishingConstants() {
+        throw new IllegalStateException("Utility class");
+    }
+
```

Comment:
We have a Lombok dependency in the project already. We can use [`@UtilityClass`](https://projectlombok.org/features/experimental/UtilityClass) annotation instead.
```suggestion
@UtilityClass
class PublishingConstants {
```